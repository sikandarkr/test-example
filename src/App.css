.a{
    background-color: red;
}
.register-page{
    /* display:flex;
    flex-direction: column; */
    width:50% !important;
    background-color:white;
    box-shadow: 0.1px 0.1px 0.1px 0.1px #888888; 
  
    /* overflow: scroll; */
}
.register-form{
   margin-left:10% !important;
   margin-right:10% !important;
   padding-top:5% !important;
 
}
.spinner-gif{
    position: absolute;  
    left: 25%;  
    right: 25%;  
    top: 25%;  
    bottom: 25%;  
    margin: auto;  
    
    height:15% !important;
    width:15% !important;
}

.register-container{
    margin-left:10% !important;
    margin-right:10% !important;
    background-color:#D3D3D3;
    border-radius: 2px !important;
   
}
.nothing{
    margin-bottom: 20px !important;
    padding-top:20px !important;
}

.submit-form{
    margin-bottom:20px !important;
}
.create-account-title{
    color:#459976 !important;
    margin-top:4% !important;
}
.navbar-top{
    background-color: #669999;
    padding:1.5% !important;
}
.register{
    background-color: blue;
    height: 100% !important;
    width:100% !important;

}
.search-icon{
    font-size:20px !important;
    background-color:white !important;
    margin-top:2% !important;
    border-left:1px;
    /* margin-left:-10px !important; */
}
.nav-dropdown{
    margin-left:5% !important;
}
.input_login_password{
    margin-left:1% !important;
    border: none !important;
    outline: none !important;
    border-radius: 5px !important;
}
.login-submit{
    background-color: #85adad;
    color:white !important;
    border-radius: 5px !important;
    border: none !important;
    border-color:gray !important;
    
}
.input-login{
    border: none !important;
    outline: none !important;
    border-radius: 5px !important;
}
.search-div{
    display: flex;
    flex-direction: row;
    width:60% !important;
    background-color: white !important;
    border-radius: 5px !important;
    height:30px !important;
}
.nav-profile{
    display: flex;
    flex-direction: row;
    justify-content: space-between;
}
.input-search{
    box-shadow: none;
    border: none !important;
    outline: none !important;
    border-radius: 5px !important;
    width:90% !important;
    height:30px !important;
}
::-webkit-input-placeholder {
    font-style: normal;
    text-transform: capitalize;
    padding-left:20px !important;
}
::-webkit-input-placeholder:focus {
    font-style: normal;
    text-transform: capitalize;
    padding-left:20px !important;
    outline: none;
}


:-moz-placeholder { /* older Firefox*/
    font-style: normal;
    text-transform: capitalize;
    padding-left:20px !important;   
}

:-moz-placeholder:focus { /* older Firefox*/
font-style: normal;
text-transform: capitalize;
padding-left:20px !important;   
outline: none !important;
}

:-moz-placeholder { /* Firefox 19+ */ 
    font-style: normal; 
    text-transform: capitalize;
    padding-left:20px !important; 
} 
:-moz-placeholder :focus{ /* Firefox 19+ */ 
font-style: normal; 
text-transform: capitalize;
padding-left:20px !important; 
outline: none !important;
} 
:-ms-input-placeholder { 
    font-style: normal;
    text-transform: capitalize;
    padding-left:20px !important;
    
}
:-ms-input-placeholder :focus{ 
font-style: normal;
text-transform: capitalize;
padding-left:20px !important;
outline: none !important;

} 