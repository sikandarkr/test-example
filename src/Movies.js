// import React,{Component} from 'react';

// class Movies extends Component{

// }

class Movies{
    api(){
        return fetch("https://facebook.github.io/react-native/movies.json").then((res)=>{
            return res.json();
        })
    }
}

export default Movies;