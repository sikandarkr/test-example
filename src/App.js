import React from 'react';
import { Route, Switch, BrowserRouter as router, Redirect } from 'react-router-dom';
import TopNavbar from '../src/component/Pages/Navbar';
import PrivateRoute from '../src/PrivateRoute';
import AuthRoute from '../src/AuthRoute';
import Register from '../src/component/Auth/Register';
import Profiles from '../src/component/Pages/Profiles'
import './App.css';
class  App extends React.Component {
  render(){
    return (
      <div className="App">
        
          <router>
            <switch>
              <Route  path="/" component={TopNavbar}/>
              <AuthRoute exact path="/login" component={Register} />
              <AuthRoute exact path="/u" component={Register} />
              <PrivateRoute exact path="/profile" component={Profiles} />
            </switch>
          </router>
      </div>
    );
  }
}
export default App;
