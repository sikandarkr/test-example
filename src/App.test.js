import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import App from './App';
import Movies from './Movies';

// it('Api testing', async () => {
//     const response = new Movies();
//     var res = await response.api();
//     expect(res.movies[0].id).toEqual("1");
// });

test("snapshot test",()=>{
  const component  = renderer.create(
    <App/>
  );
  let tree = component.toJSON();

  expect(tree).toMatchSnapshot();
})