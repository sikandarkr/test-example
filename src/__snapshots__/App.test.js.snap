// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test 1`] = `
<div
  className="App"
>
  <p>
    Hiii
  </p>
</div>
`;
